taxi_service/settings.py

```diff
@@ -0,0 +1,126 @@
+"""
+Django settings for taxi_service project.
+
+Generated by 'django-admin startproject' using Django 4.1.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-j9tj^q@c+g=btzfk#)fs6d@ailkav1j&z^vmq9aim7q^^o#&b$"
```

Comment:
keep the secret key used in production secret!  
Get this variabel as env variable.